src/pages/projects.astro

```diff
@@ -75,10 +75,10 @@ const seo = {
             </h1>
 
             <p class="dark:opacity-[92%]">
-              Ultramarine Linux is a Fedora-based Linux distribution designed to
```

Comment:
Why remove all references to Fedora?

Reply:
Felt odd to point out our upstream in a brief summary, makes more sense on the full page than here